common/index.d.ts

```diff
@@ -118,3 +118,20 @@ export interface ITeam {
   active: boolean;
   color: string;
 }
+
+declare enum Type {
+  printIssue,
+  onlineIssue,
+}
+
+/**
+ * Interface for a Issue Schema.
+ */
+export interface IIssue {
+  _id: string;
+  name: string;
+  deadlineDate: string;
+  releaseDate: string;
+  pitches: string[];
+  type: Type;
```

Comment:
Should by type string

Reply:
or the type would be the enum IssueFormatType right?

Reply:
oh never mind!